### ISSUE TITLE
Not enabling

### ISSUE THREAD

**Issue by DazaXIII** [skiselkov/BetterPushbackC#101]
Trying and failing to join the Better Pushback revolution. 
I’ve got a backup copy of XPlane installed which is clean and vanilla and Better Pushback works fine on that so I know my download and installation are correct. However on my copy of XPlane I fly with it won’t enable. Nothing in the drop down menu and clicking the checkbox of the plugin admin XP freezes for a couple of seconds then continues still without the tick in the box.   
http://i.imgur.com/CoZqcFY.jpg

I’m not really sure what I’m doing with logs but I did see this. 
  
2017-08-13 10:35:17 BetterPushback[airportdb.c:1776]: navdata error: it appears your simulator's navigation database is broken, or your simulator contains no airport scenery. Please reinstall the database and retry. 
  
I did run the installer – Update XPlane and Update Scenery Online – and still the same issue. With my fairly poor knowledge of these things I’m scratching my head as to what’s happening so any help would be much appreciated. I’ve tried v0.41 and version v0.40 with the same results 
Log attached in case there's anything in there to help diagnose a problem. 
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/1220464/Log.txt)

**Comment by skiselkov** [comment]
Gonna take a look, stand by.

**Comment by skiselkov** [comment]
Can you please check that under `Resources/default data/CIFP` you have some `.dat` files present? It should look something like this:
<img width="630" alt="shot" src="https://user-images.githubusercontent.com/3300688/29252688-f25977ee-806b-11e7-83a6-24ade26a3a66.png">
The problem that BP is reporting is that it found no airports with instrument procedures. That means that either you don't have ANY airport scenery present, or you don't have any instrument procedure data for them (which BP needs, because it automatically ignores non-instrument airfields). By default, X-Plane comes with both airport scenery and instrument procedure data, so this error basically means that your X-Plane installation is broken in some bizarre way. Kinda strange that the X-Plane updater doesn't detect this. Maybe you didn't select any scenery region to be installed and so it skipped installing the nav data? Don't know, but for now, we need to figure out if you have any of that procedure data present.

**Comment by DazaXIII** [comment]
Problem solved. I did have that folder so checked my backup copy of XPlane where I had got BP to work and it’s the same folder size and number of dat files. However just to try, I deleted my main XPlane CIFP folder and copied my backup folder over and things seem to be working fine now. 
As I’d not noticed anything wrong in XP I wonder if that may mean there were corrupted files for parts of the world I’d not been to yet so hadn’t seen any issues come to light. Whatever it was though I’ve just tested at a few different airports and Better Pushback seems to be working as it should so a big thankyou for your speedy and helpful response which is much appreciated.
